AEPMessaging/Sources/UI/ContentCards/Template/LargeImageTemplate.swift

```diff
@@ -0,0 +1,100 @@
+/*
+ Copyright 2025 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+#if canImport(SwiftUI)
+    import SwiftUI
+#endif
+
+/// A class representing a content card template with a large image layout.
+///
+/// `LargeImageTemplate` is a subclass of `BaseTemplate` and conforms to the `ContentCardTemplate` protocol.
+/// It provides a structured layout for content cards that include a large image, title, body, and optional buttons.
+/// This class is initialized with `ContentCardSchemaData`, which is used to populate the template's properties.
+/// If the required `title` is missing from the schema data, the initialization fails.
+///
+/// - Note: The `view` property is lazily initialized and represents the entire layout of the content card.
+@available(iOS 15.0, *)
+public class LargeImageTemplate: BaseTemplate, ContentCardTemplate {
+    public var templateType: ContentCardTemplateType = .largeImage
+
+    /// The title of the content card.
+    public var title: AEPText
+
+    /// The body text of the content card, if available.
```

Comment:
nit: you're probably following existing convention here, but including the `if available` for these descriptions feels redundant :)